connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSourceTask.java

```diff
@@ -50,6 +50,7 @@
     private static final Logger log = LoggerFactory.getLogger(WorkerSourceTask.class);
 
     private static final long SEND_FAILED_BACKOFF_MS = 100;
+    private static final long PAUSE_BACKOFF_MS = 100;
```

Comment:
That the pause timeout right and it can be waken before the time out. Maybe change the name to something like a timeout? Also, any specific reason to use 100ms as the timeout?